intellij-plugin-verifier/verifier-test/src/test/java/com/jetbrains/pluginverifier/tests/bytecode/BinaryClassContent.kt

```diff
@@ -0,0 +1,23 @@
+package com.jetbrains.pluginverifier.tests.bytecode
+
+import com.jetbrains.pluginverifier.verifiers.resolution.BinaryClassName
+
+internal data class BinaryClassContent(val name: BinaryClassName, val bytes: ByteArray) {
+  override fun equals(other: Any?): Boolean {
```

Comment:
Why are `equals` and `hashCode` implemented explicitly? Kotlin data classes provide them out-of-the-box. Aren't they sufficient?

Reply:
`ByteArray` does not provide `equals()`/`hashCode()`. In addition, Kotlin/IDEA suggests that array-based properties in data classes should provide these methods explicitly.